__tests__/hooks/usePostDetail.test.js

```diff
@@ -84,33 +84,33 @@ describe('usePOstDetail should run correctly', () => {
     jest.spyOn(Dimensions, 'get').mockReturnValue({width: 400, height: 800});
     const {result} = renderHook(usePostDetail);
     expect(result.current.calculationText('wide')).toEqual({
-      containerComment: 23.333333333333314,
-      containerHeight: 346.6666666666667,
-      fontSize: 25.6,
-      lineHeight: 46.93333333333333,
+      containerComment: 77.5,
+      containerHeight: 292.5,
+      fontSize: 21.6,
+      lineHeight: 39.6,
       isShortText: true
     });
     expect(result.current.calculationText('wide', 2)).toEqual({
-      containerComment: 23.333333333333314,
-      containerHeight: 346.6666666666667,
-      fontSize: 25.6,
-      lineHeight: 46.93333333333333,
+      containerComment: 77.5,
+      containerHeight: 292.5,
+      fontSize: 21.6,
+      lineHeight: 39.6,
       isShortText: true
     });
 
     expect(result.current.calculationText('wide', 1)).toEqual({
-      containerComment: 23.333333333333314,
-      containerHeight: 346.6666666666667,
-      fontSize: 25.6,
-      lineHeight: 46.93333333333333,
+      containerComment: 77.5,
+      containerHeight: 292.5,
+      fontSize: 21.6,
+      lineHeight: 39.6,
       isShortText: true
     });
 
     expect(result.current.calculationText(longText)).toEqual({
-      containerComment: -794.8,
-      containerHeight: 1164.8,
-      fontSize: 17.066666666666666,
-      lineHeight: 25.6,
+      containerComment: -461.6,
+      containerHeight: 831.6,
+      fontSize: 14.4,
+      lineHeight: 21.6,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in the `calculationText` function's return values seem to be significant. It appears that the dimensions and font sizes have been adjusted, which could potentially affect the layout and readability of the text in your application. Please ensure these changes are thoroughly tested on different screen sizes and resolutions to avoid any UI issues.
```diff
-      containerComment: 23.333333333333314,
-      containerHeight: 346.6666666666667,
-      fontSize: 25.6,
-      lineHeight: 46.93333333333333,
+      containerComment: 77.5,
+      containerHeight: 292.5,
+      fontSize: 21.6,
+      lineHeight: 39.6,
```
Also, it would be beneficial to add comments explaining why these specific values were chosen and how they were calculated. This will help future developers understand the reasoning behind these numbers and make it easier for them to maintain or modify this code.


<!-- This is an auto-generated comment by OSS CodeRabbit -->